app/src/main/java/com/google/samples/apps/sunflower/viewmodels/PlantDetailViewModel.kt

```diff
@@ -38,6 +39,7 @@ class PlantDetailViewModel(
 
     val isPlanted: LiveData<Boolean>
     val plant: LiveData<Plant>
+    val notifyToggleAddOrRemove: MutableLiveData<Boolean>
```

Comment:
there will be a problem such as screen rotation, the same msg will be redelivered, which is already outdated or irrelevant already. do refer to `SnackbarMessage` and `SingleLiveEvent` from the architecture repo.

https://github.com/googlesamples/android-architecture/blob/dev-todo-mvvm-live/todoapp/app/src/main/java/com/example/android/architecture/blueprints/todoapp/SnackbarMessage.java

Reply:
@andhie YES! Agree, I have used this pattern in other projects, too.

Reply:
@andhie @XinyueZ Cool, I will learn it.